lib/BuildConfigManager.js

```diff
@@ -103,7 +103,10 @@ class BuildConfigManager {
             });
         });
 
-        this._worker.on('close', () => callback('worker terminated'));
+        this._worker.on('close', () => {
+            callback('worker terminated');
+            clearTimeout(timeoutId);
```

Comment:
Address this issue https://github.com/bigcommerce/stencil-cli/actions/runs/2202902268, that was unveiled by bumping jest